### DISCUSSION TITLE
In Model.php line 26:

  Interface "Illuminate\Contracts\Queue\QueueableEnti
  ty" not found

### DISCUSSION THREAD

**Discussion by deser1** [laravel/framework#43376] [Q&A]
In Model.php line 26:

  Interface "Illuminate\Contracts\Queue\QueueableEntity" not found

How repair this problem before update composer 9.19 => 9.21
Please help!

**Comment by michael-rubel** [comment]
Hi @deser1 

Update your `composer` to the latest version and try to run these commands:
- `composer clear-compiled`
- `composer dump-autoload`

You can as well clear the Laravel's internal caches:
- `php artisan optimize:clear`

---

**Reply by deser1** [reply]
Ok. I need test this tip. :)
Best regards
